src/features/views/day/day-carousel/DayItem.tsx

```diff
@@ -30,10 +30,20 @@ const DayItem = ({ date, active, onClick }: DayItemProps) => {
     const invalid = isAfter(date, new Date())
     const mappedRating = rating && Math.round((rating / 5) * 6 + 1)
 
-    return { dayLetter, dayNumber, count, invalid, mappedRating }
+    // DorHakim
+    const today = isToday(date)
+
+    // Or implementing ourselves
+    // const today =
+    //   date.toLocaleDateString('he') === new Date().toLocaleDateString('he')
+    //     ? true
+    //     : false
```

Comment:
If we were to implement it ourselves, it's not a good implementation.
`toLocaleDateString('he')` creates a string specifically in Hebrew. Why would we want to compare strings, and evben ones with Hebrew characters, when we can compare numbers efficiently?
A better implementation would be comparing `getYear`, `getMonth` and `getDate`.
Either way, the library handles it so you may delete the commented code.